homework_24/tests/tests.py

```diff
@@ -0,0 +1,98 @@
+"""PYTEST TEST CASES"""
+
+import time
+import pytest
+from homework_24.pages.login_page import LoginPage
+from homework_24.pages.contact_list_page import ContactListPage
+from homework_24.pages.contact_details_page import ContactDetailsPage
+from homework_24.pages.edit_contact_page import EditContactPage
+from homework_24.pages.add_contact_page import AddContactPage
+import homework_24.test_data.constants as const
+from log_dir.log_setup import logger
+
+
+@pytest.mark.create_new_contact
+def test_create_new_contact(browser):
+    """
+    Test the creation of a new contact.
+    """
+    logger.info("TEST 1: CREATE NEW CONTACT. Executing...")
+    page_object = LoginPage(browser)
+    page_object.complete_login(const.EMAIL, const.PASSWORD)
+    page_object = ContactListPage(browser)
+    page_object.click_add_contact()
+    page_object = AddContactPage(browser)
+
+    page_object.complete_add_new_contact(const.FIRST_NAME, const.LAST_NAME,
+                                         const.BIRTHDATE, const.EMAIL_ADDRESS,
+                                         const.PHONE, const.STREET1,
+                                         const.STREET2, const.CITY,
+                                         const.STATE_PROVINCE,
+                                         const.POSTAL_CODE, const.COUNTRY)
+    page_object = ContactListPage(browser)
+    assert page_object.locate_contact_row() is not None
+    logger.info("TEST 1: Executed")
+
+
+@pytest.mark.edit_contact
+def test_edit_contact(browser):
+    """
+    Test the editing of an existing contact.
+    """
+    logger.info("TEST 2: EDIT CONTACT. Executing...")
+    page_object = LoginPage(browser)
```

Comment:
Better names: 
```suggestion
    login_page = LoginPage(browser)
    ...
    contact_list_page = ContactListPage(browser)
```
and other....

Reply:
Replaced with "current_page" everywhere.